dashboard/src/shared/utils.ts

```diff
@@ -125,3 +125,11 @@ export function getPluginPackageName(plugin?: Plugin | string) {
     }
   }
 }
+
+export function getPluginsRequiringSA(): string[] {
+  return [PluginNames.PACKAGES_FLUX, PluginNames.PACKAGES_KAPP];
+}
+
+export function getPluginsSupportingRollback(): string[] {
+  return [PluginNames.PACKAGES_HELM];
+}
```

Comment:
Extracting the logic of which plugins are for certain capabilities, to DRY-up the code.

Reply:
Excellent - much better to have it all in the one place for when we want to replace it with the plugin-provided items themselves.